app/views/static_pages/support_us.html.erb

```diff
@@ -0,0 +1,123 @@
+<%= title('Support the Project') %>
+
+<div id="support-us">
+  <section class='page-container'>
+    <div class='grid grid-cols-12'>
+      <div class='col-span-12 md:col-start-3 md:col-span-8 space-y-8'>
+        <div class='text-center'>
+          <h1 class='page-heading-title'>Support the Project</h1>
+          <p class='max-w-prose mx-auto prose prose-gray dark:prose-invert text-gray-500 dark:text-gray-400'>
+            From the very beginning, The Odin Project has been committed to providing a world-class and completely free coding curriculum for anyone and everyone eager to learn.
+            <strong>With your generous donations</strong>, we can continue to inspire thousands of aspiring developers, irrespective of their background or financial status.
+          </p>
+        </div>
+      </div>
+    </div>
+
+    <div class="mt-8 text-center">
+      <%= link_to 'Donate now!', 'https://opencollective.com/theodinproject/donate?amount=5', class: 'button button--primary text-base px-12 py-4', target: '_blank' %>
```

Comment:
small nit: do you think donate now button in support us page also deserves the arrow icon similar to the footer donate now button?

Reply:
in that case

![giphy](https://github.com/TheOdinProject/theodinproject/assets/85733202/7ec2c57d-e5e1-4e5b-8d74-e05ba1ec9535)